developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,116 @@
+```
+:::
 
-## Initial Account Setup
+## Creating an account
 
-An Aptos account is referenced by an account address, which is a 32-byte value. The account address is derived from the initial public verification key(s) for that account. Specifically, the account address is the 32-byte of the SHA-3 256 cryptographic hash of the initial public verification key(s) concatenated with a signature scheme identifier byte. The Aptos Blockchain supports two signature schemes: [Ed25519](/reference/glossary#ed25519) and MultiEd25519 (for multi-signature transactions). The account's private key is necessary for signing transactions.
+When a new account is requested by the user, for example by using the [Aptos SDK](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html), then the following cryptographic steps are executed to create the account. Below is a conceptual view of these steps:
 
-Each account also stores a `sequence_number`, which represents the next transaction sequence number to prevent replay attacks of transactions. This is initialized to `0` at account creation time.
+- Start by generating a new private key, public key pair.
+- From the user get the preferred signature scheme for the account: If the account should use a single signature or if it should require multiple signatures for signing a transaction. 
+- Combine the public key with the user's signature scheme to generate a 32-byte authentication key. 
+- Initialize the sequence number to 0. The sequence number represents the next transaction sequence number, to prevent replay attacks of transactions. Both the authentication key and the sequence number are stored in the account as an initial account resource. 
+- Create the 32-byte account address from the initial authentication key. 
 
-## Authentication Keys
+From now on, the user should use the private key for signing the transactions with this account. 
 
-Each account stores an authentication key. This authentication key enables account owners to rotate their private key(s) associated with the account without changing the address that hosts their account. During rotation, the authentication key is updated based upon the newly-generated private, public key-pair(s).
+## Account address
 
-### Single signer authentication
+During the new account creation process, a 32-byte authentication key comes to exist first. This authentication key is then returned as it is as the 32-byte account address. Hence, **only initially** the 32-byte authentication key will be the same as the 32-byte account address. However, the authentication key may subsequently change, for example, when you generate a new pair of private key, public keys, to rotate the keys. But the account address will not change. After an account is created, the account address will remain unchanged even though the private key, public key and authentication keys may change. There is nothing called changing the address of the existing account. 
```

Comment:
The first two sentences need some rearrangements.

I think before you mention only initially, you should mention account key rotation.